demo/chbench/grafana/conf/dashboards/.gitattributes

```diff
@@ -0,0 +1 @@
+*.json linguist-generated=true
```

Comment:
I think perhaps we want `-diff` here rather than `linguist-generated=true`, so that the diffs are also suppressed when you run `git diff` manually.

Reply:
@quodlibetor is the one who changes these the most often, so we should let him make the call on whether he'd rather see them in `git diff` or not.

Reply:
what does `-diff` do? does it make it just print a summary?

I _do_ look at the diffs.

Reply:
It will cause `git diff` locally to print `Binary files differ` instead of the actual diff.

Since you do look at the diffs, I'll land it as-is, unless Nikhil expresses further concerns.

Reply:
👍